### PR TITLE
Join tables using pk and fk mechanism

### PR DESCRIPTION
**Pull request by ezraermy**
This pull request creates a database using PostgreSQL
The implementation includes the following changes:
- Query multiple tables
- Use primary key & foreign key mechanisms for joining tables.
- Prepare queries that answer analytical questions.
- Screenshots:
-  join_table_queries1
-  join_table_queries2
- join_table_queries3

## Other essential requirements:
- Use Gitflow.
- Documented in a professional way.

### CODE REVIEW

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @ezraermy 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] A new table with the name `vets` , `specializations` and `visits` have been added👍.
- [x] Data have been inserted into the vets, specialization and visits table👍.
- [x] Queries have been  as requested👍.



## Required Changes ♻️

- [ ] Nice job so far, but just as required, let's add a screenshot to our pull request showing each commands and their results to show how we meet the requirements.👍
image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
Hi, @Ibizugbe Thank you for your time reviewing my pr and giving me a feedback. 

You haven't seen the screenshot files I provided in the pr list, namely join_tables, join_table_queries1, join_table_queries2, and join_table_queries3. All those are screenshots of the result of my queries for this assignment. l would love your suggestion on which query result is not presented in my repo. Thanks again.

**Comment by Ibizugbe** [comment]
Hello @ezraermy👍
I can see the images were listed but they are not being dsipaled because you didn't upload them😉. Kindly upload the files to your PR description😉 When this is done, we will be able to see the images on your PR description👍. Do you get understand?

File: queries.sql [hunk 1]
```diff
@@ -84,3 +84,69 @@ JOIN owners o ON a.owner_id = o.id
 GROUP BY o.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
+
```
Comment (danmainah) [inline queries.sql:87]: - [ ] @ezraermy great work adding the queries to your pull request, however, they are not showing, below is the screenshot of how the pull request looks currently
image

**Comment by ezraermy** [comment]
Screenshot from 2023-05-11 16-35-35
@Ibizugbe All screenshots are there. Maybe you checked wrong branch (dev), My current active branch is join_table.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ezraermy , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow was correctly used
✔️  All queries added correctly

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibizugbe** [comment]
Hello @ezraermy. The images are present in your directory or repository and I saw that👍. But that is not the instruction that was given.  The instruction states that you add them to your Pull request description👍. This way, it becomes easy for the reviewer to see the images showing how your transactions worked👍. I suggest you edit your Pull request description and add all the images 😉.

**Comment by ezraermy** [comment]
@danmainah Thank you for the feedback. I attached all files as per your and the previous reviewer's request.

**Comment by danmainah** [comment]
> @danmainah Thank you for the feedback. I attached all files as per your and the previous reviewer's request.

@ezraermy I still cant see them maybe check how you are uploading them

**Comment by ezraermy** [comment]
Screenshot from 2023-05-11 17-00-14
@danmainah Can you please check one more time?

**Comment by danmainah** [comment]
> Screenshot from 2023-05-11 17-00-14 @danmainah Can you please check one more time?

@ezraermy This is a link to the screenshot, the project requirement requires the screenshot to be on the pull request and not a link

**Comment by ezraermy** [comment]
@danmainah Thank you for your patience and helpful feedback. I wasn't submitted this way before which is why I am a bit confused. I hope you can see them now.

**Comment by danmainah** [comment]
> @danmainah Thank you for your patience and helpful feedback. I wasn't submitted this way before which is why I am a bit confused. I hope you can see them now.

@ezraermy Yes they are visible now, great work.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ezraermy 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Querry screenshots ok

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**